spikingjelly/activation_based/triton_kernel/flexsn/template.py

```
@@ -270,13 +402,110 @@ def get_flexsn_forward_kernel(
     return kernel_exe
 
 
+def get_flexsn_forward_final_state_kernel(
+    core_str: str,
+    core_name: str,
+    info: FlexSNInfo,
+    verbose: bool = False,
+):
+    hash = f"{core_name[-8:]}_{BASE_AUTOTUNE_TAG}"
```

Ruff (0.15.11)

[error] Variable `hash` is shadowing a Python builtin

(A001)